src/contracts/ZapperLidoARM.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+// Contracts
+import {Ownable} from "./Ownable.sol";
+
+// Interfaces
+import {IWETH} from "./Interfaces.sol";
+import {IERC20} from "./Interfaces.sol";
+import {ILidoARM} from "./Interfaces.sol";
+
+contract ZapperLidoARM is Ownable {
```

Comment:
I don't think we need the Zapper to be `Ownable`. I see that is used for `rescueERC20` but there shouldn't be any tokens in this contract. I see the OETH Zapper doesn't have this.

If we do want the Zapper to be Ownerable, we need to decide who the owner should be. Options are:
1. 5/8 multisig
2. Strategist 2/9 multisig
3. ARM Defender Relayer
My preference would be the Strategist

@DanielVF what do you think?

Reply:
I completely agree, it was just in case someone mistakenly send token to it (like people sending wETH to wETH contract). 
I have no problem removing it.

However if we want to keep it, I think Strategist or Relayer is enough.